packages/forms/src/UIForm/fields/MultiSelectTag/MultiSelectTagWidget.scss

```diff
@@ -0,0 +1,62 @@
+$tf-datalist-items-background-color: $white !default;
+$tf-datalist-items-shadow-color: $shadow !default;
+$tf-typeahead-input-width: 135px !default;
```

Comment:
where the 135 comes from ?

Reply:
That's a value defined here to determine the min width of the input.